cms/envs/bok_choy.py

```diff
@@ -77,6 +77,15 @@
 YOUTUBE['TEST_URL'] = "127.0.0.1:{0}/test_youtube/".format(YOUTUBE_PORT)
 YOUTUBE['TEXT_API']['url'] = "127.0.0.1:{0}/test_transcripts_youtube/".format(YOUTUBE_PORT)
 
+FEATURES['ENABLE_COURSEWARE_INDEX'] = True
+SEARCH_ENGINE = "search.tests.mock_search_engine.MockSearchEngine"
+# Path at which to store the mock index - can remove disable=invalid-name
+# once global settings are set to disable this error and
+# disable=no-value-for-parameter follow the pattern above
```

Comment:
Ah yes, disable=invalid-name has been added to the top of this file, so this comment is out of date - unfortunately, no-value-for-parameter is very much the same - all the other settings that get their value from calls using the / path operator also have this problem.